src/Microsoft.AspNetCore.Testing/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -4,14 +4,17 @@
 
   <PropertyGroup>
     <Description>Various helpers for writing tests that use ASP.NET Core.</Description>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
We need the two here because of the following: https://github.com/aspnet/Testing/blob/dev/src/Microsoft.AspNetCore.Testing/xunit/FrameworkSkipConditionAttribute.cs#L35-L53